src/renderer/families/cosmos/UndelegationFlowModal/fields/Amount.js

```diff
@@ -28,7 +28,10 @@ export default function AmountField({
   const { t } = useTranslation();
   const unit = getAccountUnit(account);
 
-  const initialAmount = useMemo(() => (validator ? validator.amount : BigNumber(0)), [validator]);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const initialAmount = useMemo(() => (validator ? validator.amount : BigNumber(0)), [
```

Comment:
i'm not sure why we need a memo for this (if there is a business logic related to it, it worth documenting, but also i feel we use memo wrong if it's the case as react don't guarantee you memo won't get recalculated 🤔 ). there is a overuse of memo in our app.

Reply:
i updated the logic here maybe its more clear that way we need to update the "initialAmount" variable each time the source validator is changed a sideEffect is indeed a better way to do things